rpm/qmf-qt5.spec

```diff
@@ -56,6 +58,11 @@ Patch24:  0024-Adjust-qmflist-for-missing-bits-in-5.6.patch
 Patch25:  0025-Revert-loadRelax.patch
 Patch26:  0026-Revert-Set-PLUGIN_CLASS_NAME-in-plugin-.pro-files.patch
 Patch27:  0027-Revert-Bump-version-to-6.0.0-since-we-build-against-.patch
+Patch28:  0028-Protect-service.patch
```

Comment:
This should be 0028-Protect-service-pointer-access.patch

Reply:
Ah, thanks, morrected.